prow/cmd/crier/main.go

```diff
@@ -160,7 +160,7 @@ func main() {
 	controllers := []*crier.Controller{}
 
 	if o.gerrit {
-		gerritReporter, err := reporter.NewReporter(o.cookiefilePath, o.projects)
```

Comment:
should this really be a commit ...? /shrug

Reply:
🙃 I cannot see which commit you are commenting on... but... I'll squash :-)